web/src/components/profile/approved-researcher/TrainingCertificate.tsx

```diff
@@ -30,15 +32,30 @@ export default function TrainingCertificate() {
   const [isPDF, setIsPDF] = useState<boolean>(false);
   const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
 
+  function base64Encode(file: File): Promise<string | null> {
```

Comment:
why nest?

Reply:
You mean inside the `TrainingCertificate` function? Theres no strict need I guess. Happy to move it outside the function 👍